flycheck.el

```diff
@@ -8630,6 +8631,16 @@ See URL `http://slim-lang.com'."
           line-end))
   :modes slim-mode)
 
+(flycheck-define-checker slim-lint
+  "A SLIM linter
+
+See URL `https://github.com/sds/slim-lint'."
+  :command ("slim-lint" "--reporter=checkstyle"
+            source-original)
+  :error-parser flycheck-parse-checkstyle
+  :modes slim-mode
+  :next-checkers ((info . slim)))
```

Comment:
That would seem correct to me as well but I wasn't sure how the precedence works and how it works if you don't have one or the other installed.